.gitattributes

```diff
@@ -1,3 +1,5 @@
 scripts/rff/raw_data/prism/daily/county/** filter=lfs diff=lfs merge=lfs -text
+scripts/rff/raw_data/prism/daily/4km/** filter=lfs diff=lfs merge=lfs -text
 scripts/rff/raw_data/us_vectors/** filter=lfs diff=lfs merge=lfs -text
-scripts/rff/*.csv filter=lfs diff=lfs merge=lfs -text
\ No newline at end of file
+scripts/rff/historical_weather_variability/4km/*.csv filter=lfs diff=lfs merge=lfs -text
```

Comment:
to confirm, how big are the input/output files?

If they're in GBs, then could you please consider copying it to our cloud bucket:
`gsutil -m cp <files> gs://datcom-csv/rff/`

I have given permissions to rff-datacommons@googlegroups.com

Thanks!

Reply:
Will do, I'll let you know when it's done. All other comments should be addressed

Reply:
All the source-data should be uploaded to GCS

Reply:
To confirm, have you tried running this one the whole dataset? Any expectations on how long it might run, how much CSV output it might produce?

Reply:
All the historical 4km and county-level data should be uploaded to GCS already. It took about 20 minutes. I'm not sure how big the CSV output will be but hopefully we can just check how much data has been uploaded to the gs://datcom-csv/rff/ folder and estimate from there.

Reply:
I see that you have uploaded the original dataset.  I was asking about the generated CSV.  From your response, I take it that you've not run the script on the entire dataset.

I will merge this PR, and look into running it.